app/app.js

```diff
@@ -39,9 +41,16 @@
 				matchesCtrl.territories = response.data.data;
 			});
 		}
+		
+		this.sendOrders = function(matchId, empireId, ordersText){
+			$window.alert(server + '/api/rest/matches/'+matchId+'/empires/'+empireId+'/orders?order_str='+escape(ordersText));
```

Comment:
Whenever I duplicate a string like that in two lines, I ALWAYS screw it up.. Like, will modify one and not the other.  Now I always assign the string to a variable, then output it and use it.  So many hours lost to that.  They look identical here though.